makefile

```diff
@@ -36,6 +38,11 @@ ifndef NOWERROR
    CFLAGS += -Werror
 endif
 
+ifdef LOG
+   DEFINES += -DLOG_COMMS
```

Comment:
Actually, I'd much prefer this was a command line option instead of a build option.

Reply:
Right, might be a future implementation